workflow.md

```diff
@@ -240,3 +256,57 @@ document, this is to be attached as comment.
 
 All tasks on current sprint are to be assigned an estimation and properly
 transition according to real state
+
+
+
+## Release managament
+
+This section describes our release practices.
+
+### Fields in YT
+
+We have a number of special-purpose fields in YT for release management:
+
+* Affected versions
+   + For bug reports/testcase failures, should contain a list of versions, for which issue was approved to reproduce.
+* Affected builds
+   + For bug reports/testcase failures, should contain a list of builds, for which issue was approved to reproduce.
+* Target versions
+   + Version, within which we are to deliver the patch associated with issue.
+   
+### Release processing
+
+Following steps should be executed to release version `X = Maj.Min.Fix`:
+
+1. Check issues assigned to target version `X` are all in *Wait for build* state
+
+   Issues should be moved to different version iff no commits were pushed to branch corresponding to `X`.
+
+   In case some commits were pushed, some work remained, issue should be split to two (one assigned to next version).
+   Commit revert may also be an option.
+
+2. Extract list of commits from git commit history, i.e. all commits:
+   * From tag of version `Maj.Min.(Fix-1)` if `Fix > 0`.
+   * From tag of version `Maj.(Min - 1).0` if `Fix == 0`.
+
+3. Compare list of issues from Git history and list of issues for version `X`
+
+   To make them conform:
+   * Change version for issues in YT
+   * Revert some commits in branch
+
+4. Trigger build in CI
+
+5. Once CI built successfully:
+   1. Put tag in branch for minor version.
+      If `Fix == 0`, also put tag in branch for major version
+   2. Mark version in YT as released
+   3. Add artifact links from CI to version description
+
+6. Issues from version to be set status *Waiting for test* and assigned to QA team lead.
+
+7. QA team to perform consise testing of version (release note testing or regression testing, depending on case)
+
+8. QA team to approve no blockers discovered (i.e. only minor issues/glitches remain)
+
+9. Release to be published to end user (and delivered by CSL update mechanism)
```

Comment:
I don't think CSL update mechanism should be mentioned here. AFAIK, documents in this repository are about all projects involved into CSL. However, if someone updates something like `node-sketch` it doesn't have to be delivered to end user.